packages/slice-machine/src/pages/index.tsx

```diff
@@ -1,16 +1,8 @@
 import { type FC } from "react";
 
 import { CustomTypesTablePage as CustomTypesTablePageTemplate } from "@/features/customTypes/customTypesTable/CustomTypesTablePage";
-import { PageTypeOnboarding } from "@/features/customTypes/PageTypeOnboarding";
-import { useIsEmptyProject } from "@/hooks/useIsEmptyProject";
```

Comment:
just to be sure, this one is still being used elsewhere? (to remove it otherwise)

Reply:
No, actually not being used anywhere else, looked like a useful hook to leave 😄

Reply:
Removed

Reply:
Correction: It's still being used by the existing onboarding guide, which I will remove next.
I started to remove it already, so I didn't see any other usages.